test/distributed/test_store.py

```diff
@@ -743,6 +744,31 @@ def test_wait(self, i):
 
 instantiate_parametrized_tests(TestMultiThreadedWait)
 
+class InitPgWithUvStore(TestCase):
+    def test_with_url_param(self):
+        port = common.find_free_port()
```

Comment:
i think we should just use port 0 since there is a race condition where the port could be used before it is used by the store.

Reply:
Funny story, I tried that but it doesn't work, we can't pass 0 as the port when using rendezvous.

Reply:
zero evals to false and in rendezvous we do a `if not port`.